packages/slider/sp-slider.ts

```diff
@@ -10,11 +10,14 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 import { Slider } from './src/Slider.js';
+import { SliderHandle } from './src/SliderHandle.js';
 
 customElements.define('sp-slider', Slider);
+customElements.define('sp-slider-handle', SliderHandle);
 
 declare global {
     interface HTMLElementTagNameMap {
         'sp-slider': Slider;
+        'sp-slider-handle': SliderHandle;
```

Comment:
I'm happy to change if you think it is worth adding in an `sp-slider-handle.ts`.

Reply:
Will reserve judgement till the very end, but possibly.

Reply:
Speaking through this a little:

Having this together means that if you want to use an `sp-slider` element but for some reason want to use a custom/extended `sp-slider-handle` element (possibly one that closes over some of the complexities you've shown possible in the three handle demos), a consumer would _have_ to extend and use a new element name. This works fine because you leverage `this.host.querySelectorAll('[slot="handle"]')` in the model resolver, so I don't think we have a strong need to move this. It is a little outside of other usage, none of the other paired elements register in a single file, but I'm not sure it's that big of a deal.

If you were to read this and decide "let's move it to `sp-slider-handle.ts`!", do take into account that the new file would need to be added to the `exports` map in `package.json` were you to do that.

Reply:
Ok, broke down and created `sp-slider-handle.ts`